test/test-readability.js

```diff
@@ -60,7 +60,7 @@ describe("Test page", function() {
         host: "fakehost",
         prePath: "http://fakehost",
         scheme: "http",
-        pathBase: "http://fakehost/test"
+        pathBase: "http://fakehost/test/"
```

Comment:
For http/https, it seems so. Not for chrome://, but I don't think we care about this case here.